proposals/csharp-12.0/collection-expressions.md

```diff
@@ -229,6 +231,21 @@ ImmutableArray<int> ia =
 
     Here, if `b` is false, it is not required that any value actually be constructed for the empty literal since it would immediately be spread into zero values in the final literal.
```

Comment:
```suggestion
    Here, if `b` is false, it is not required that any value actually be constructed for the empty collection expression since it would immediately be spread into zero values in the final literal.
```